.changeset/warm-ducks-dig.md

```diff
@@ -0,0 +1,6 @@
+---
+'@spectrum-web-components/link': patch
+'@spectrum-web-components/styles': patch
+---
+
+Remove unnecessary system theme references to reduce complexity for components that don't need the additional mapping layer.
```

Comment:
Do we want to mention the CSS link package update? Is that necessary or helpful to list out in the changeset at all?

Reply:
I don't think we need to mention that the dependency updated explicitly, just wanted to make sure I captured the _what_ of the change for the SWC logs.